Formula/minio-mc.rb

```diff
@@ -0,0 +1,27 @@
+class MinioMc < Formula
+  desc "ls, cp, mkdir, diff and rsync for filesystems and object storage"
+  homepage "https://github.com/minio/mc"
+  url "https://github.com/minio/mc/archive/RELEASE.2016-07-13T21-46-05Z.tar.gz"
+  version "20160713214605"
+  sha256 "611444a66ea3d2cc8fbb821147bcd3995c1956ef2aa9d62cd35f6d07f3f972cf"
+  head "https://github.com/minio/mc.git"
+  conflicts_with "midnight-commander"
```

Comment:
Please leave an empty line before this line and it's nicer if you provide a reason as a second argument (see also other formula that use this), e.g.:

```ruby
conflicts_with "midnight-commander", :because => "Both install a `mc` binary"
```

Please also add a similar `conflicts_with` line to the `midnight-commander` formula and include it in your PR. (The conflicts need to be declared in all involved formulae.)